sdk/eventhubs/azure-messaging-eventhubs-perf/src/main/java/com/azure/messaging/eventhubs/perf/standalone/EventHubsConsumerClientPerf.java

```diff
@@ -0,0 +1,136 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.messaging.eventhubs.perf.standalone;
+
+import com.azure.messaging.eventhubs.EventHubClientBuilder;
+import com.azure.messaging.eventhubs.EventHubConsumerAsyncClient;
+import com.azure.messaging.eventhubs.models.EventPosition;
+import org.apache.commons.cli.CommandLineParser;
+import org.apache.commons.cli.DefaultParser;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.Options;
+import org.apache.commons.cli.HelpFormatter;
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.ParseException;
+import org.apache.qpid.proton.InterruptException;
+
+import java.util.concurrent.CountDownLatch;
+
+/**
+ * Standalone app to run performance test for consumer client.
+ * TODO: Port to framework test once it supports event based perf testing.
+ */
+public class EventHubsConsumerClientPerf {
+    private static final String EVENTHUB_NAME = System.getenv("EVENTHUB_NAME");;
+// Settings copied from
+// https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-faq#how-much-does-a-single-capacity-unit-let-me-achieve
+    private static final int BYTES_PER_MESSAGE = 1024;
+
+    /**
+     * Runs the EventHub consumer client performance test
+     *
+     * @param args The arguments used to run the performance test.
+     * @throws InterruptedException when thread is interrupted during execution.
+     */
+    public static void main(String[] args) throws InterruptedException {
+        Options options = new Options();
+        Option clientsOption = new Option("c", "clients", true, "Number of client instances");
+        options.addOption(clientsOption);
+        Option partitionsOption = new Option("p", "partitions", true, "Number of partitions");
+        options.addOption(partitionsOption);
```

Comment:
Not sure why this option is required. The number of partitions for an event hub is defined on the service side.